### PR TITLE
Update ci for integration tests

### PR DESCRIPTION
**Pull request by reshmabidikar**
Update shared workflow to run Java integration tests

### CODE REVIEW

**Review by pierre — COMMENTED** [review]
I suspect this is for the grouping plugin?

Since it's the only plugin with such Java integration tests, I think we should skip that new Java step by default. Maybe `test-goal` has a default empty value, and if it's not specified, we skip that step (only the grouping plugin would specify that argument for now)?

**Comment by reshmabidikar** [comment]
> Since it's the only plugin with such Java integration tests, I think we should skip that new Java step by default. Maybe test-goal has a default empty value, and if it's not specified, we skip that step (only the grouping plugin would specify that argument for now)?

Makes sense, I've made modifications for this.

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -219,7 +219,9 @@ jobs:
             sleep 1
           done
       - name: Run Java Integration Tests
-        run: ./mvnw --errors --batch-mode --show-version -P${{ matrix.integration-test-profile }} ${{ inputs.maven_args_tests }} ${{ inputs.test-goal }}
+        if: inputs.test-goal == 'test verify'
```
Comment (pierre) [inline .github/workflows/integration_test.yml:222]: How about `if: inputs.test-goal != '' ` instead? That way, we can specify custom goals when we want to.

**Review by pierre — APPROVED** [review]

File: .github/workflows/integration_test.yml (hunk 1 above)
Comment (reshmabidikar) [reply .github/workflows/integration_test.yml]: Fixed

**Pull request merged**